src/services/user.ts

```diff
@@ -0,0 +1,10 @@
+import { User } from '../types/User';
+import usersFromServer from '../api/users';
+
+export function getAllUsers(): User[] {
+  return usersFromServer;
+}
```

Comment:
this function does nothing. you can just import a variable where you need. Try to not overthink)